documentation/operators/src/nodes/maintenance.md

```diff
@@ -205,33 +205,79 @@ This lets your operating system know it's ok to reload the service configuration
 
 ## Moving a node
 
-In case of a need to move a node from one machine to another and avoiding to lose the delegation, here are few steps how to do it.
-
-The following examples transfers a Mix Node (in case of other nodes, change the `mixnodes` in the command for the `<NODE>` of your desire.
-
-* Pause your node process.
+In case of a need to move a Nym Node from one machine to another and avoiding to lose the delegation, here are few steps how to do it.
 
 Assuming both machines are remote VPS.
 
-* Make sure your `~/.ssh/<YOUR_KEY>.pub` is in both of the machines `~/.ssh/authorized_keys` file
+* Make sure your `~/.ssh/<YOUR_KEY>.pub` is in both of the servers `~/.ssh/authorized_keys` file
 * Create a `nym-nodes` folder in the target VPS. SSH in from your terminal and run:
 
 ```sh
 # in case none of the nym configs was created previously
 mkdir ~/.nym
 
-#in case no nym Mix Node was initialized previously
+#in case no nym Nym Node was initialized previously
```

Comment:
do we do initialised or initialized throughout the docs?

Reply:
Max is an "s" vanguard here. So is the entire organi**s**ation of the docs <3<3